src/pykeen/training/slcwa.py

```diff
@@ -65,7 +65,6 @@ def _create_training_data_loader(
                 drop_last=drop_last,
                 negative_sampler=self.negative_sampler,
                 negative_sampler_kwargs=self.negative_sampler_kwargs,
-                sampler=sampler,
```

Comment:
@mberr heads up on this - it seems like this was a bug before but it just wasn't bothering anyone until the updates here